templates/config/config.cue

```diff
@@ -91,21 +91,47 @@ import (
 #Percent:  string & =~"^(100|[1-9][0-9]?)%$"
 
 #Monitoring: {
-	// Enable Prometheus monitoring
-	enabled: *false | bool
-	serviceMonitor: {
-		// Enable Prometheus ServiceMonitor monitoring instead of the prometheus shim
-		enabled:            *false | bool
-		prometheusInstance: *"default" | string
-		targetPort:         *"http-metrics" | int | string
-		path:               *"/metrics" | string
-		interval:           *"60s" | #Duration
-		scrapeTimeout:      *"30s" | #Duration
-		labels?:            timoniv1.#Labels
-		annotations?:       timoniv1.#Annotations
-		honorLabels:        *false | bool
-		endpointAdditionalProperties?: {[string]: string}
-	}
+	// Enable Prometheus monitoring for the cert-manager controller to use with the
+	// Prometheus Operator. If this option is enabled without enabling `prometheus.servicemonitor.enabled` or
+	// `prometheus.podmonitor.enabled`, 'prometheus.io' annotations are added to the cert-manager Deployment
+	// resources. Additionally, a service is created which can be used together
+	// with your own ServiceMonitor (managed outside of this Helm chart).
+	// Otherwise, a ServiceMonitor/ PodMonitor is created.
+	enabled: *false | true
+	// The type of monitoring to enable, can be one of "ServiceMonitor", "PodMonitor" or "Annotations"
+	type: "ServiceMonitor" | "PodMonitor" | *"Annotations"
+	// Create a PodMonitor to add cert-manager to Prometheus
+	enabled: *false | true
```

Comment:
Need to fix this, the comment should note the type field instead of different fields and the second enabled should be removed.